cmd/skipper/main.go

```diff
@@ -128,10 +128,12 @@ const (
 	kubernetesNamespaceUsage        = "watch only this namespace for ingresses"
 
 	// OAuth2:
-	oauthURLUsage            = "OAuth2 URL for Innkeeper authentication"
-	oauthCredentialsDirUsage = "directory where oauth credentials are stored: client.json and user.json"
-	oauthScopeUsage          = "the whitespace separated list of oauth scopes"
-	oauth2TokeninfoURLUsage  = "sets the default tokeninfo URL to query information about an incoming OAuth2 token in oauth2Tokeninfo filters"
+	defaultOAuthTokeninfoTimeout = 100 * time.Millisecond // Not sure if this makes sense
+	oauthURLUsage                = "OAuth2 URL for Innkeeper authentication"
+	oauthCredentialsDirUsage     = "directory where oauth credentials are stored: client.json and user.json"
+	oauthScopeUsage              = "the whitespace separated list of oauth scopes"
+	oauth2TokeninfoURLUsage      = "sets the default tokeninfo URL to query information about an incoming OAuth2 token in oauth2Tokeninfo filters"
+	oauth2TokeninfoTimeoutUsage  = "sets the default tokeninfo request timeout duration"
```

Comment:
Let's add some info what a value of 0 means - no timeout, stdlib default (how much?), etc.

Reply:
Is it enough to add the timeout value of 1500ms from `defaultOAuthTokeninfoTimeout `?